recipes/hmftools-esvee/meta.yaml

```diff
@@ -12,13 +12,13 @@ source:
 
 build:
   noarch: generic
-  number: 3
+  number: 4
   run_exports:
     - {{ pin_subpackage("hmftools-esvee", max_pin="x.x") }}
 
 requirements:
   run:
-    - openjdk >=8
+    - openjdk ==11.0.1
```

Comment:
Those pins are too strict? Why do you need exactly this version? can it be >=11?

Reply:
I've now allow openjdk versions 8 to 17.

This version restriction was to prevent a multithreading crash related to later versions of java as mentioned in this [github issue](https://github.com/elastic/elasticsearch/issues/106987).

I'm now going to test to make sure this crash doesn't occur with openjdk 17